shardingsphere-underlying/shardingsphere-common/src/main/java/org/apache/shardingsphere/underlying/common/metadata/schema/RuleSchemaMetaDataLoader.java

```diff
@@ -161,4 +167,32 @@ private TableMetaData decorate(final String tableName, final TableMetaData table
         }
         return result;
     }
+    
+    private Map<String, SchemaMetaData> asyncLoadSchemaMetaDataMap(final Map<String, DataSource> dataSourceMap, final int maxConnectionCount,
```

Comment:
this method is the same with org.apache.shardingsphere.sql.parser.binder.metadata.schema.SchemaMetaDataLoader， could we merge and keep one method?

Reply:
I will use ShardingSpehre's thread pool to transform SchemaMetaDataLoader.asyncLoad() method and RuleSchemaMetaDataLoader.asyncLoadSchemaMetaDataMap() method, How's that

Reply:
> I will use ShardingSpehre's thread pool to transform SchemaMetaDataLoader.asyncLoad() method and RuleSchemaMetaDataLoader.asyncLoadSchemaMetaDataMap() method, How's that

Reuse is a way?